src/api/app/views/webui/package/submit_request.html.haml

```diff
@@ -1,57 +1,54 @@
-.modal.fade#submit-request-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'submit-request-modal-label', hidden: true } }
-  .modal-dialog.modal-dialog-centered{ role: 'document' }
-    .modal-content
-      = form_tag({ controller: 'package', action: 'submit_request' }, method: 'post') do
-        .modal-header
-          %h5.modal-title#submit-request-modal-label Create Submit Request
-        .modal-body
-          %p
-            Do you want to submit #{package_link package, rev: revision}?
+:ruby
+  @pagetitle = 'Create submit request'
```

Comment:
We click on `Submit Package` to go on this page, which then has the title `Create submit request` while the breadcrumbs are also `Submit Package`.

This might be clearer with an image:
![consistent-page-title](https://user-images.githubusercontent.com/1102934/81814945-b0c32c00-9529-11ea-9531-49bccbfa4ffb.png)

I suggest using `Submit Package` everywhere, even for the name of the view file.